man/man8/zfs.8

```diff
@@ -3035,6 +3037,40 @@ Print verbose information about the stream and the time required to perform the
 
 .sp
 .ne 2
+.mk
+.na
+\fB-o\fR \fIproperty\fR=\fIvalue\fR
+.ad
+.sp .6
+.RS 4n
+Sets the specified property as if the command \fBzfs set\fR \fIproperty\fR=\fIvalue\fR was invoked at the same time the received dataset was created from the non-incremental send stream or updated from the incremental send stream.
+.sp
+Any editable ZFS property can also be set at receive time. Set-once properties bound to the received data, such as \fBnormalization\fR and \fBcasesensitivity\fR, cannot be set at receive time even when the datasets are newly created by zfs receive.
+.sp
+Multiple \fB-o\fR options can be specified. An error results if the same property is specified in multiple \fB-o\fR or \fB-x\fR options.
```

Comment:
How about: "The `-o` option may be specified multiple times, for different properties."